test/jdk/tools/jpackage/share/AppImagePackageTest.java

```diff
@@ -61,4 +65,37 @@ public static void test() {
             cmd.removeArgumentWithValue("--input");
         }).addBundleDesktopIntegrationVerifier(false).run();
     }
+
+    @Test
+    public static void testEmpty() {
+        final String name = "EmptyAppImagePackageTest";
+        final String imageName = name + (TKit.isOSX() ? ".app" : "");
+        Path appImageDir = TKit.workDir().resolve(imageName);
```

Comment:
I'd suggest to use Tkit.createTempDirectory() to create temp directory. The function will create unique directory every time it is called allowing to run the test multiple times and have fresh empty directory in every run.